clients/unshaded/src/main/java/tachyon/client/file/FileInStream.java

```diff
@@ -217,7 +223,7 @@ private void checkAndAdvanceBlockInStream() throws IOException {
           mCurrentCacheStream =
               mContext.getTachyonBlockStore().getOutStream(currentBlockId, -1, null);
         } catch (IOException ioe) {
-          // TODO(yupeng): Maybe debug log here.
+          LOG.info("Failed to get BufferedBlockOutStream: " + ioe.getMessage());
```

Comment:
Not sure if info is good here, perhaps warn? Also I think you can specify the exception as the second parameter instead of adding it to the message.